ivy/utils/backend/handler.py

```diff
@@ -243,7 +243,7 @@ def current_backend(*args, **kwargs):
     return importlib.import_module(_backend_dict[implicit_backend])
 
 
-def _set_backend_as_ivy(
```

Comment:
I prefer `_set_backend_as_ivy` as it's tends to be more informative than `_modify_module_symbols`

Reply:
since it was used recursively it does not only modify ivy but all the sub modules, so I though that `_set_backend_as_ivy` does not tell the whole purpose of the function.
does that make sense ?

Reply:
Yeah sounds good, but I think `symbols` could also be a bit misleading, also `set` would be a better prefix than `modify`. Probably let's settle on `_set_module_backend_as_ivy`, what do you think?

Reply:
I noticed also in the signiture
![image](https://github.com/unifyai/ivy/assets/100947451/9835709e-e82f-4d90-9ee6-bd2eaa36c273)
that the second argument is `target` and in the usage:
![image](https://github.com/unifyai/ivy/assets/100947451/6298f311-fb3c-407e-999f-36f904b511e1)
we pass ivy as a second argument so it should be something like: _set the terget as backend_ so the name maybe is `_set_module_as(terget=ivy, backend=backend)` or we can settle on `_set_ivy_as_backend`

Reply:
@vedpatwardhan, a gentle reminder 😄

Reply:
Thanks for the reminder, sorry missed out on this commit in the last review 😅
Yeah I think `_set_module_backend` would be the semantically preferable name because it indicates that we're setting the backend of the module while `_set_backend_as_ivy` indicates that we're setting `ivy` as a backend which isn't the case.

Reply:
thanks for refactoring this!